src/components/NavBar.tsx

```diff
@@ -160,6 +161,16 @@ export default function NavBar({ organization, setShowNavBar }: NavBarProps): JS
               id='nurseries'
             />
           )}
+          {isEnabled('Tracking V1') && (
+            <NavItem
```

Comment:
Guess there's no icon since we will move it under Locations. cc @joshbuchea in case it needs to be rebased into visual_updates branch.